trifling.py

```diff
@@ -889,3 +889,11 @@ def weather(bot, trigger):
 
     except:
         bot.say("The earth is on fire 🌎🔥")
+
+@sopel.module.commands('yeezy')
+def yeezy(bot, trigger):
+    try:
+        res = requests.get('https://api.kanye.rest')
```

Comment:
- add a timeout
- add an user agent
- check for 2XX status code
- check response type
- validate json schema

```python3
headers = {"User-Agent": "Wow!"}
resp = requests.get('https://api.kanye.rest', headers=headers, timeout=3)
resp.raise_for_status()
blob = resp.json()
if "quote" not in blob or not isinstance(blob["quote"], str):
    raise Exception("malformed response")
bot.say(blob["quote"])
```

Reply:
sacre bleu! am I in a code review right now?!

Reply:
I'm afraid so!!

Reply:
done, sir.